### PR TITLE
Fixed TuneIn

### PR DESCRIPTION
**Pull request by DarkVillager**
- Added a new check to check if there's an error.
- Changed Duration to match changed code so it works.
- Changed paused checking for different text in the Element.

image
image

### CODE REVIEW

File: websites/T/TuneIn/presence.ts [hunk 1]
```diff
@@ -82,10 +87,16 @@ presence.on("UpdateData", async () => {
           delete data.endTimestamp;
         }
 
-        presence.setActivity(data);
+        if (data.details === null) {
```
Comment (SlowLife1661) [inline websites/T/TuneIn/presence.ts:90]:
```suggestion
        if (!data.details) {
```

**Review by Slowlife01 — DISMISSED** [review]

**Review by Bas950 — CHANGES_REQUESTED** [review]
run prettier

File: websites/T/TuneIn/presence.ts [hunk 2]
```diff
@@ -14,78 +14,84 @@ presence.on("UpdateData", async () => {
...
-        if (paused) {
-          delete data.startTimestamp;
-          delete data.endTimestamp;
-        }
+          (data.smallImageKey = paused ? "pause" : "play"),
+            (data.smallImageText = paused
+              ? (await strings).pause
+              : (await strings).play),
+            (data.startTimestamp = timestamps[0]),
+            (data.endTimestamp = timestamps[1]);
```
Comment (Bas950) [inline websites/T/TuneIn/presence.ts:82]: use ; not (),

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/T/TuneIn/presence.ts (hunk 2 above)
Comment (DarkVillager) [reply websites/T/TuneIn/presence.ts:82]: Thats a prettier thing?

Comment (Bas950) [reply websites/T/TuneIn/presence.ts:82]: yea its because you use . not ;

Comment (Bas950) [reply websites/T/TuneIn/presence.ts:82]: image
those should be ;

File: websites/T/TuneIn/presence.ts [hunk 3]
```diff
@@ -14,78 +14,84 @@ presence.on("UpdateData", async () => {
...
+
+          data.smallImageKey = paused ? "pause" : "play";
+          data.smallImageText = paused
             ? (await strings).pause
-            : (await strings).play),
-          (data.startTimestamp = timestamps[0]),
-          (data.endTimestamp = timestamps[1]);
+            : (await strings).play;
+          data.startTimestamp = timestamps[0];
+          data.endTimestamp = timestamps[1];
```
Comment (Slowlife01) [inline websites/T/TuneIn/presence.ts:82]:
```suggestion
          data.endTimestamp = timestamps.pop();
```
```suggestion
          [, data.endTimestamp] = timestamps;
```

**Review by Bas950 — APPROVED** [review]

**Review by Slowlife01 — APPROVED** [review]

**Pull request merged**